### PR TITLE
test: Enable updates-testing for fedora-24

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-24

**Comment by cockpituous** [comment]
Image creation for fedora-24 in process on host37-rack09.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-24-b7031917d97ab0610dd847c22104abafb4014d19/

**Comment by cockpituous** [comment]
Image creation done: https://github.com/cockpituous/cockpit/commits/refresh-fedora-24-b7031917d97ab0610dd847c22104abafb4014d19

File: test/images/scripts/fedora-24.setup [hunk 1]
```diff
@@ -69,7 +69,12 @@ echo 'NETWORKING=yes' > /etc/sysconfig/network
 useradd -c Administrator -G wheel admin
 echo foobar | passwd --stdin admin
 
+# HACK - we have to enable updates-testing twice because the upgrade
+# might switch it off again.
```
Comment (stefwalter) [inline test/images/scripts/fedora-24.setup]: Is there a bug filed for this?

**Comment by stefwalter** [comment]
Is there a reason why we're doing this? Can we just install one necessary package from updates-testing, with a HACK indicating the situation?

**Comment by mvollmer** [comment]
> Is there a reason why we're doing this? Can we just install one necessary package from updates-testing, with a HACK indicating the situation?

I thought the idea is to use updates-testing for fedora-24 in general until we switch fedora-testing to 24.

**Comment by stefwalter** [comment]
> I thought the idea is to use updates-testing for fedora-24 in general until we switch fedora-testing to 24.

Okay, if you've discussed and decided that then I guess it makes sense. I imagine you also talked about just updating fedora-testing?

But that logic probably needs to be put into the commit message, right?

**Comment by mvollmer** [comment]
Let's not do this, and just follow fedora.

**Pull request closed without merging**